.github/workflows/ci.yml

```diff
@@ -133,14 +138,17 @@ jobs:
           wget https://github.com/KhronosGroup/OpenCL-SDK/releases/download/v2023.04.17/OpenCL-SDK-v2023.04.17-Win-x64.zip
           unzip -j OpenCL-SDK-v2023.04.17-Win-x64.zip OpenCL-SDK-v2023.04.17-Win-x64/lib/OpenCL.lib
 
+      - name: Version suffix (for release only)
+        id: version
+        run: echo "::set-output name=suffix::${{ github.ref_type == 'tag' && '-' || ''}}${{ github.ref_type == 'tag' && github.ref || ''}}"
```

Comment:
::set-output is deprecated. There is an alternative way to do the same, please check this GH blog post: https://github.blog/changelog/2022-10-11-github-actions-deprecating-save-state-and-set-output-commands/